tests/mapper/test_graph_generator.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Data61, CSIRO
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from stellargraph.core.graph import *
+from stellargraph.mapper.graph_generator import GraphGenerator, GraphSequence
+
+import numpy as np
+import pytest
+from ..test_utils.graphs import example_graph_random, example_hin_1
+
+
+class Test_GraphGenerator:
+
+    graphs = [
+        example_graph_random(feature_size=4, n_nodes=6),
+        example_graph_random(feature_size=4, n_nodes=5),
+        example_graph_random(feature_size=4, n_nodes=3),
+    ]
+
+    graphs_nx = [
```

Comment:
Small nit: this and the next two are only used in a single test each; maybe they can be just defined within the relevant test rather than scoped globally?